speculation-rules/prerender/workers.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<title>Same-origin prerendering can access localStorage</title>
+<meta name="timeout" content="long">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/common/utils.js"></script>
+<script src="resources/utils.js"></script>
+<body>
+<script>
+
+setup(() => assertSpeculationRulesIsSupported());
+
+promise_test(async t => {
+  const {exec, activate} = await create_prerendered_page(t);
+  const workerURL = new URL('resources/worker-post-timeOrigin.js', location.href).toString();
+  await exec(workerURL => {
+    window.worker = new Worker(workerURL);
+    window.waitForWorker = new Promise(resolve => worker.addEventListener('message', e => {
+        resolve({
+          prerendering: document.prerendering,
+          activationStart: performance.getEntriesByType('navigation')[0].activationStart,
+          workerLoad: performance.getEntriesByName(workerURL)[0].startTime,
+          workerStart: e.data});
+    }), workerURL);
+  }, workerURL);
+
+  await new Promise(resolve => t.step_timeout(resolve, 300));
```

Comment:
Comment about the purpose of this? I assume it's so that there is a detectable delta between the timestamps reported?

Reply:
Added